crates/swc_common/src/plugin/mod.rs

```diff
@@ -1,7 +1,7 @@
 pub mod diagnostics;
 pub mod metadata;
-#[cfg(feature = "plugin-base")]
-#[cfg_attr(docsrs, doc(cfg(feature = "plugin-base")))]
+#[cfg(feature = "__plugin")]
```

Comment:
I think this name is misleading

Reply:
I feel overall it's a bit complex with namings, while I choose the short path as I think we may get rid of all of this once verification is done. If we're going to keep this permanent, it'd be probably better to organize bit more.